realtime/_async/client.py

```diff
@@ -7,47 +7,46 @@
 from functools import wraps
 from math import floor
 from typing import Any, Callable, Dict, List, Optional
+from urllib.parse import urlencode, urlparse, urlunparse
 
 import websockets
+from websockets.asyncio.client import ClientConnection, connect
 
-from ..exceptions import NotConnectedError
 from ..message import Message
 from ..transformers import http_endpoint_url
 from ..types import (
     DEFAULT_TIMEOUT,
     PHOENIX_CHANNEL,
-    Callback,
+    VSN,
     ChannelEvents,
-    T_ParamSpec,
-    T_Retval,
 )
 from ..utils import is_ws_url
 from .channel import AsyncRealtimeChannel, RealtimeChannelOptions
 
-logger = logging.getLogger(__name__)
-
 
-def ensure_connection(func: Callback):
```

Comment:
Since we are removing this functionality which was only used internally would this cause any breaking change?

Reply:
It is only a breaking change if the caller is handling the `NotConnectedError` raised by this wrapper, which is unlikely.

With its removal, methods which requires a connection, now just returns silently, which better aligns with the behavior from JS.